### PR TITLE
DS-721 | @ericbakenhus | Add results count

### PR DESCRIPTION
**Pull request by ericbakenhus**
# READY FOR REVIEW

# Summary
- Add results count
- Adjust flow of surrounding elements

# Review By (Date)
- Retro

# Criticality
- Low

# Review Tasks

## Setup tasks and/or behavior to test

1. Look at the [deploy preview](https://deploy-preview-884--stanford-alumni.netlify.app/events/discovery/)
2. Verify that the results count is visible above the events
3. Verify there is visibly hidden additional page-related data for screen readers

# Associated Issues and/or People
- DS-721

### CODE REVIEW

File: src/components/events-discovery/components/StatusHeader/StatusHeader.jsx [hunk 1]
```diff
@@ -1,40 +1,50 @@
...
-/**
- * @type {React.FC<Props>}
+ * @type {React.FC<{}>}
  * @returns {React.ReactElement}
  */
-export const StatusHeader = ({ hitsPerPage }) => {
-  const { currentRefinement, nbHits } = usePagination();
-  const { items } = useHits();
+export const StatusHeader = () => {
+  const { nbHits, hitsPerPage, nbPages, page } = useStats();
```
Comment (ericbakenhus) [inline src/components/events-discovery/components/StatusHeader/StatusHeader.jsx:9]: Switching from `usePagination` and `useHits` to `useStats`.

File: src/components/events-discovery/components/Event/Event.jsx [hunk 2]
```diff
@@ -71,20 +70,11 @@ export const Event = ({
...
 
-  // Find current UTC date/time
-  const currentUTCDate = new Date();
-  const luxonCurrent = DateTime.fromJSDate(currentUTCDate);
-
-  // If the current date/time is after the event end date/time, don't render the card
-  if (!title || !start || !end || luxonCurrent > luxonEnd) {
-    return null;
-  }
-
```
Comment (ericbakenhus) [inline src/components/events-discovery/components/Event/Event.jsx:82]: Ah, right. This just makes the events empty. Removed.

File: src/components/events-discovery/eventsDiscovery.jsx [hunk 3]
```diff
@@ -103,7 +110,13 @@ const EventsDiscovery = () => (
...
     insights
   >
-    <Configure hitsPerPage={hitsPerPage} />
+    <Configure
+      hitsPerPage={hitsPerPage}
+      // Don't let any expired events slip through
+      filters={`endTimestamp > ${DateTime.now()
+        .startOf('day')
+        .toUnixInteger()}`}
+    />
```
Comment (ericbakenhus) [inline src/components/events-discovery/eventsDiscovery.jsx:119]: Last line of defense to keep old events from popping up.

Comment (sherakama) [reply src/components/events-discovery/eventsDiscovery.jsx:119]: As we build then let time pass, this makes complete sense to me.

**Review by sherakama — COMMENTED** [review]
GTG

**Pull request merged**